packages/react-ui-ag/src/Listings/DesktopListing.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+import { ListingComponents } from '@rentpath/react-ui-core'
+import Listing from './Listing'
+
+const buttonPropTypes = PropTypes.shape({
```

Comment:
can we move all custom prop type in separate file ?

Reply:
Why is that necessary?

Reply:
Oh since `buttonPropTypes` is the same in `DesktopListing` and `MobileListing`?

Reply:
I'd say this is a "nice to have" refactor, but I'll still give you a +1 without it.

Reply:
Yeah, I don't think this is truly necessary and once we move to TypeScript it'll be irrelevant as they will be in the definition files, but I do like the thought process @paudyalhari 👍